doc/classes/MainLoop.xml

```diff
@@ -5,40 +5,53 @@
 	</brief_description>
 	<description>
 		[MainLoop] is the abstract base class for a Godot project's game loop. It is inherited by [SceneTree], which is the default game loop implementation used in Godot projects, though it is also possible to write and use one's own [MainLoop] subclass instead of the scene tree.
-		Upon the application start, a [MainLoop] implementation must be provided to the OS; otherwise, the application will exit. This happens automatically (and a [SceneTree] is created) unless a main [Script] is provided from the command line (with e.g. [code]godot -s my_loop.gd[/code], which should then be a [MainLoop] implementation.
+		Upon the application start, a [MainLoop] implementation must be provided to the OS; otherwise, the application will exit. This happens automatically (and a [SceneTree] is created) unless a [MainLoop] [Script] is provided from the command line (with e.g. [code]godot -s my_loop.gd[/code] or the "Main Loop Type" project setting is overwritten.
 		Here is an example script implementing a simple [MainLoop]:
-		[b]FIXME:[/b] No longer valid after DisplayServer split and Input refactoring.
 		[codeblock]
+		class_name CustomMainLoop
 		extends MainLoop
 
 		var time_elapsed = 0
-		var keys_typed = []
 		var quit = false
 
 		func _initialize():
```

Comment:
I guess if this is meant to follow the style guide, an extra line should be added to have two lines between member declarations and method declarations.

I'm not sure we ever had a consensus on using double line separations in short code snippets of the class reference though. (I think we actually erred on the side of not doing it in the classref, I even see things like `Control.xml` where there's no separation line at all for short examples.)

Reply:
I think @NathanLovato said that we should use single lines for separation in the class reference, but two lines in projects, so that we can fit more content in the documentation and so that it's easy to see where the functions are in more complex scripts in projects. I can't find exactly this statement, but see this discussion: https://github.com/godotengine/godot-docs/issues/3613

Reply:
The style guide didn't change, it's still two-line between functions. Vnen suggested one-line for cases where we just have a few lines of code or many snippets.

> I still argue for the exception on small snippets in the docs, given that's often incomplete code [...]

For example in cases like this, that look off with two lines:

```gd
var test = 3


func _ready():
    print(test)
```

As opposed to

```gd
var test = 3

func _ready():
    print(test)
```

Reply:
Let me know if you'd like to add a note for one-line separation in the class reference.

Reply:
I also tend to agree that one-line separation in the class reference is better (as an exception to the style guide that applies to longer snippets / demos).

Reply:
I'll add a note box to the gdscript style guide.

Reply:
Done https://github.com/godotengine/godot-docs/pull/4384